e2e/suite/upgrade_test.go

```diff
@@ -0,0 +1,67 @@
+package suite_test
+
+import (
+	"time"
+
+	"github.com/rancher/system-upgrade-controller/e2e/framework"
+	upgradeapiv1 "github.com/rancher/system-upgrade-controller/pkg/apis/upgrade.cattle.io/v1"
+)
+
+var _ = Describe("Upgrade", func() {
```

Comment:
everything else in this commit exists to run these three simple tests. will add more once this lands on master